# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344448628 25200
#      Wed Aug 08 10:57:08 2012 -0700
# Node ID 9087c2816136109f0a87f0622d10c40e892338a4
# Parent  f635710b309080f05fb13b483bafbd8f7fd9ce3a
src/Makefile.am: Remove unnecessary -I$(srcdir) in AM_CPPFLAGS.

* src/Makefile.am: Remove unnecessary -I$(srcdir) in AM_CPPFLAGS.

diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -25,17 +25,16 @@ AM_CPPFLAGS = \
   -I$(top_srcdir)/libcruft/misc \
   -I../liboctave -I$(top_srcdir)/liboctave \
   -Ioctave-value -I$(srcdir)/octave-value \
   -Iparse-tree -I$(srcdir)/parse-tree \
   -Ioperators -I$(srcdir)/operators \
   -Iinterp-core -I$(srcdir)/interp-core \
   -Iinterpfcn -I$(srcdir)/interpfcn \
   -Icorefcn \
-  -I$(srcdir) \
   -I../libgnu -I$(top_srcdir)/libgnu
 
 AUTOMAKE_OPTIONS = subdir-objects
 
 octlib_LTLIBRARIES = liboctinterp.la
 
 if AMCOND_BUILD_COMPILED_AUX_PROGRAMS
 bin_PROGRAMS = \
